Tests/RestAPI/OktaAPIRequestTests.swift

```diff
@@ -135,8 +135,16 @@ class OktaAPIRequestTests : XCTestCase {
         let httpResponse = HTTPURLResponse(url: url, statusCode: 400, httpVersion: nil, headerFields: nil)!
         let data = "{\"errorCode\":\"\(errorCode)\"}".data(using: .utf8)!
         
-        req.handleResponse(data: data, response: httpResponse)
+        req.handleResponse(data: data, response: httpResponse, error: nil)
         
         wait(for: [exp], timeout: 1.0)
     }
+    
+    func testInjectHttpClient() {
```

Comment:
You also need to write a test that confirms that `run` works fine without injected delegate